rfc/README.md

```diff
@@ -6,6 +6,9 @@ We write an RFC to _communicate intent and get feedback_ before we start buildin
 
 Whenever your team is making a decision that could affect the platform, write the details and get feedback from others on the Platform. Anyone - including product managers, UX, research, and engineers - is encouraged to create RFCs in order to solicit ideas in preparation of a decision direction.
 
+There is no formal requirement to present an RFC at the [Platform Architecture Working Group meeting](https://vfs.atlassian.net/wiki/spaces/ECP/pages/1872363553/Platform+Architecture+Working+Group), but authors may _request_ to present there. Depending on the subject and/or scope of an RFC the Platform Architecture WG _may_ request author(s) to present either by its own authority or because it has been petitioned to do so.
```

Comment:
> There is no formal requirement to present an RFC

This is excellent clarification, thanks.

Reply:
Clarification is good. I do struggle with the concept of a lot of time around formality, etc. for something that isn't required. Of course, for VFS teams, this would be a bit more complicated to enforce but for VSP teams we absolutely could.